src/random_pyp.py

```diff
@@ -0,0 +1,189 @@
+from bot import bot, commands, datetime, discord, json, keywords, pymongo, requests, tasks, time, typing
+import random
+import pyshorteners
+from data import CIE_IGCSE_SUBJECT_CODES, CIE_OLEVEL_SUBJECT_CODES, CIE_ALEVEL_SUBJECT_CODES, ciealsubjectsdata, cieigsubjectsdata, cieolsubjectsdata
+
+@bot.slash_command(name="random_pyp", description="gets a random CAIE past year paper.")
+async def random_pyp(interaction: discord.Interaction, 
+                     programme: str = discord.SlashOption(name="directories", description= "IGCSE, OLEVELS or ALEVELS?", choices=["IGCSE", "O-Level", "A-Level"], required=True),
+                     subject_code: str = discord.SlashOption(name="subject_code", description="please enter the subject code", required=True),
+                     paper_number: str = discord.SlashOption(name="paper_no", description= "Enter a paper number", required=True)):
+
+            #PAPER_INFORMATION
+            insert_codes = ["0410", "0445", "0448", "0449", "0450", "0454", "0457", "0460", "0471", "0500", "0501", "0502", "0503", "0504", "0505", "0508", "0509", "0513", "0514", "0516", "0518", "0538", "9609"]
+            YEARS = ["2018", "2019", "2020", "2021", "2022", "2023"]
+            LAT_SES = ["m", "s"]
+            PAPER_VARIANT = ["1", "2", "3"]
+            SESSIONS = ["s", "w", "m"]
+
+
+            #USAGE OF THE RANDOM VARIABLE
+            ranyear = random.choice(YEARS)
+            ranvar = random.choice(PAPER_VARIANT)
+            ranses = random.choice(SESSIONS)
+            ses = random.choice(LAT_SES)
+
+            #EMPTY_VARIABLES
+            sesh = ""
+            paper_variant = ""
```

Comment:
I believe the `paper_variant` and `level` variables aren't used anywhere. Can they be removed?